BaseFilter/sections/specific.txt

```diff
@@ -18721,6 +18721,8 @@ usgamer.net##.advert-container
 dailymail.co.uk##.ad-wrapper
 !+ NOT_OPTIMIZED
 mail.yahoo.com##[data-test-id^="pencil-ad"]
+upgradeguy.com##a[href="https://upgradeguy.link/IP-Vanish-DEAL"]:has(img)
```

Comment:
`:has `only recently become natively supported, so it would be more reliable:
`upgradeguy.com##a[href="https://upgradeguy.link/IP-Vanish-DEAL"] > img`